app/stylesheets/photon.min.less

```diff
@@ -30,7 +30,7 @@ input[type=number]::-webkit-inner-spin-button,input[type=number]::-webkit-outer-
 input[type=search]::-webkit-search-cancel-button,input[type=search]::-webkit-search-decoration{-webkit-appearance:none}
 fieldset{border:1px solid silver;margin:0 2px;padding:.35em .625em .75em}
 legend{border:0;padding:0}
-*{cursor:default;-webkit-box-sizing:border-box;box-sizing:border-box}
+*{-webkit-box-sizing:border-box;box-sizing:border-box}
```

Comment:
Omfg thank you. Why did we not do that sooner.

Reply:
(Photon is a hangover from a really early iteration and it's basically been an angry monkey on our back for...forever.)